foodkept/Startup.cs

```diff
@@ -54,9 +54,9 @@ public void ConfigureServices(IServiceCollection services)
                 config.LoginPath = "/Login";
             });
 
-            services.AddIdentityCore<ApplicationUser>(options => options.SignIn.RequireConfirmedAccount = true)
+            /*services.AddIdentityCore<ApplicationUser>(options => options.SignIn.RequireConfirmedAccount = true)
                 .AddEntityFrameworkStores<ShopContext>()
-                .AddTokenProvider<DataProtectorTokenProvider<ApplicationUser>>(TokenOptions.DefaultProvider);
+                .AddTokenProvider<DataProtectorTokenProvider<ApplicationUser>>(TokenOptions.DefaultProvider);*/
 
```

Comment:
just interested why it is commented

Reply:
I had to comment this while developing, otherwise I could not log in with random accounts and test. Accidentally commited.